src/main/java/org/carlspring/strongbox/janusgraph/graph/JanusGraphConfig.java

```diff
@@ -19,14 +16,17 @@
 {
 
     @Bean
-    public Builder janusGraphFactory(CassandraDaemon cassandra,
-                                     CassandraEmbeddedProperties cassandraEmbeddedProperties)
+    JanusGraph janusGraph(GremlinServer gremlinServer)
```

Comment:
@fuss86 the first thing is that we shouldn't expose `JanusGraph` as a `@Bean`, it should be like it was before.
In terms of JPA `JanusGraph` is more like `EntityManager` and `Builder` is like `EntityManagerFactory`

And the second thing is that this is probably fine to use `GremlinServer` for graph explorer but we shouldn't use it within the application itself.

Reply:
ok, regarding the "first" part please ignore it. I've looked again and probably `JanusGraph` instance assumed to be thread safe and can be the `EntityManagerFactory` :)
I was just confused with that `JanusGraph` interface extends `Transaction` interface, but now I see that the implementation is `ThreadLocal` based and seems thread safe.

The "second"  still valid, so we shouldn't make `JanusGraph` depends on `GremlinServer`.

Reply:
done